docs/pictures/zesong/NewWS.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
```

Comment:
<img width="961" alt="Screenshot 2020-10-29 at 9 23 14 AM" src="https://user-images.githubusercontent.com/46109197/97513945-7d396880-19c8-11eb-9def-b66f607e1c83.png">

Hi I feel that some sequence diagrams might be a little too complicated (based on examples on the website). Perhaps repetitive function calls to general classes can be excluded, focusing only on the specific command classes.

Reply:
I agree with @thatseant that the sequence diagrams are little too complicated. If the other details are equally important, is it possible for them to be drawn separately instead?